docs/resources/Soundboard.md

```diff
@@ -0,0 +1,107 @@
+# Soundboard Resource
+
+Users can play soundboard sounds in voice channels, triggering a Voice Channel Effect Send Gateway event for users connected to the voice channel.
+
+There is a set of [default sounds](#DOCS_RESOURCES_SOUNDBOARD/list-default-soundboard-sounds) available to all users. Soundboard sounds can also be [created in a guild](#DOCS_RESOURCES_SOUNDBOARD/create-guild-soundboard-sound); users will be able to use the sounds in the guild, and Nitro subscribers can use them in all guilds.
+
+Soundboard sounds in a set of guilds can be retrieved over the Gateway using [Request Soundboard Sounds](#DOCS_TOPICS_GATEWAY_EVENTS/request-soundboard-sounds).
+
+### Soundboard Sound Object
+
+###### Soundboard Sound Structure
+
+| Field      | Type                                            | Description                                                               |
+|------------|-------------------------------------------------|---------------------------------------------------------------------------|
+| name       | string                                          | the name of this sound                                                    |
+| sound_id   | snowflake                                       | the id of this sound                                                      |
+| volume     | double                                          | the volume of this sound, from 0 to 1                                     |
+| emoji_id   | ?snowflake                                      | the id of this sound's custom emoji                                       |
+| emoji_name | ?string                                         | the unicode character of this sound's standard emoji                      |
+| guild_id?  | snowflake                                       | the id of the guild this sound is in                                      |
+| available  | boolean                                         | whether this sound can be used, may be false due to loss of Server Boosts |
+| user?      | [user](#DOCS_RESOURCES_USER/user-object) object | the user who created this sound                                           |
+
+### Sound Files
+
+A soundboard sound can be retrieved in MP3 or Ogg format at the URL:
+
+```
+https://cdn.discordapp.com/soundboard-sounds/{sound_id}
+```
+
+## Send Soundboard Sound % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/send-soundboard-sound
+
+Send a soundboard sound to a voice channel the user is connected to. Fires a Voice Channel Effect Send Gateway event.
```

Comment:
```suggestion
Send a soundboard sound to a voice channel the user is connected to. Fires a [Voice Channel Effect Send](#DOCS_TOPICS_GATEWAY_EVENTS/voice-channel-effect-send) Gateway event.
```
The gateway event could get a link back to the soundboard sound object as well (commenting this again, since the visibility on the other comment isn't great / it's apparently outdated)...